### DISCUSSION TITLE
[Feature Request]: led settings

### DISCUSSION THREAD

**Discussion by cracky22** [meshtastic/firmware#5180] [Ideas]
### Platform

NRF52, ESP32

### Description

on the T1000-E disable heartbeat LED but have an option for enable notification LED **AND enable charging status LED**

**Comment by garthvh** [comment]
You can enable the notification led in external notifications, charging led's are generally not addressable

**Reply by cracky22** [reply]
I know about the external notification, but the Seeed Tracker T1000-E no longer shows the charging status when the heartbeat LED is deactivated...
